package.json

```diff
@@ -19,12 +19,13 @@
     "@eslint/eslintrc": "^3.3.1",
     "@eslint/js": "^9.26.0",
     "@vitest/eslint-plugin": "^1.2.1",
+    "babel-plugin-react-compiler": "^1.0.0",
```

Comment:
does it work if you put this in the `synapse-react-client` package.json instead of here? this is not really needed by all projects, so I would prefer to install it only where it is used, if possible